go-controller/pkg/ovn/controller/services/loadbalancer.go

```diff
@@ -136,13 +218,23 @@ func EnsureLBs(nbClient libovsdbclient.Client, service *corev1.Service, existing
 		mapLBDifferenceByKey(removeLBsFromGroups, existingGroups, wantGroups, blb)
 	}
 
-	ops, err := libovsdbops.CreateOrUpdateLoadBalancersOps(nbClient, nil, existinglbs...)
+	ops, err := libovsdbops.CreateOrUpdateLoadBalancersOps(nbClient, nil, toNBLoadBalancerList(existinglbs)...)
+	if err != nil {
+		return err
+	}
+
+	ops, err = libovsdbops.CreateLoadBalancersOps(nbClient, ops, toNBLoadBalancerList(newlbs)...)
 	if err != nil {
 		return fmt.Errorf("failed to create ops for ensuring update of service %s/%s load balancers: %w",
 			service.Namespace, service.Name, err)
 	}
 
-	ops, err = libovsdbops.CreateLoadBalancersOps(nbClient, ops, newlbs...)
+	ops, err = svcCreateTemplateVarOps(nbClient, ops, toNBTemplateList(newlbs))
+	if err != nil {
+		return err
+	}
+
+	ops, err = svcCreateOrUpdateTemplateVarOps(nbClient, ops, toNBTemplateList(existinglbs))
```

Comment:
I don't think we need this split. We should just use `svcCreateOrUpdateTemplateVarOps`. The variables should either not exist or if they do, have old values? And the existence of the chassis row itself records hos no relation to the existence of the LB.

Reply:
Ok.  I changed it like you suggested.